radio/src/lua/api_model.cpp

```diff
@@ -413,6 +415,9 @@ static int luaModelInsertInput(lua_State *L)
       else if (!strcmp(key, "curveValue")) {
         expo->curve.value = luaL_checkinteger(L, -1);
       }
+      else if (!strcmp(key, "carryTrim")) {
+        expo->carryTrim= lua_toboolean(L, -1);
```

Comment:
space before "="

Reply:
right, I didn't see it.
cf our coding rules